addons/im_livechat/static/tests/embed/unread_messages.test.js

```diff
@@ -143,6 +123,6 @@ test("focus on unread livechat marks it as read", async () => {
     );
     await contains(".o-mail-ChatWindow-counter", { text: "1" });
     await contains(".o-mail-Message", { text: "Are you there?" });
-    await focus(".o-mail-Composer-input");
+    await click(".o-mail-Composer-input");
```

Comment:
Don't know whether the change from `focus()` to `click()` is important, what matters is focus state but in practice this is mostly done with a `click` so I don't mind this change anyway.